dbt/README.md

```diff
@@ -13,7 +13,7 @@ section below for examples using these profiles.
 To setup the `dbt` project, simply install the PUDL `conda` environment as normal,
 then run the following commands from this directory.
 
-```
+```bash
```

Comment:
Adding languages to the code fencing since my linter was complaining about it, and it allows syntax highlighting.